code/modules/mob/living/simple_animal/guardian/guardian.dm

```diff
@@ -0,0 +1,895 @@
+	var/magic_fluff_string = " You draw the Coder, symbolizing bugs and errors. This shouldn't happen! Submit a bug report!"
+	var/tech_fluff_string = "BOOT SEQUENCE COMPLETE. ERROR MODULE LOADED. THIS SHOULDN'T HAPPEN. Submit a bug report!"
+	var/bio_fluff_string = "Your scarabs fail to mutate. This shouldn't happen! Submit a bug report!"
+	var/admin_fluff_string = "URK URF!"//the wheels on the bus...
+
+/mob/living/simple_animal/hostile/guardian/Life() //Dies if the summoner dies
+	..()
+	if(summoner)
+		if(summoner.stat == DEAD)
+		//	src << "<span class='danger'>Your summoner has died!</span>"
+		//	visible_message("<span class='danger'><B>The [src] dies along with its user!</B></span>")
+		//	summoner.visible_message("<span class='danger'><B>[summoner]'s body is completely consumed by the strain of sustaining [src]!</B></span>")
+		//	for(var/obj/item/W in summoner)
+			//	if(!summoner.unEquip(W))
+			//		qdel(W)
+			//summoner.dust()
+			//ghostize()
+			//qdel(src)
+	//else
+			src << "<span class='danger'>Your summoner has died!</span>"
+			visible_message("<span class='danger'><B>The [src] dies along with its user!</B></span>")
+			ghostize()
+			qdel(src)
+	if(summoner)
+		if (get_dist(get_turf(summoner),get_turf(src)) <= range)
+			return
+		else
+			src << "You moved out of range, and were pulled back! You can only move [range] meters from [summoner.real_name]"
+			visible_message("<span class='danger'>The [src] jumps back to its user.</span>")
+			Recall()
+
+/mob/living/simple_animal/hostile/guardian/Move() //Returns to summoner if they move out of range
+	..()
+	if(summoner)
+		if (get_dist(get_turf(summoner),get_turf(src)) <= range)
+			return
+		else
+			src << "You moved out of range, and were pulled back! You can only move [range] meters from [summoner.real_name]"
+			visible_message("<span class='danger'>The [src] jumps back to its user.</span>")
+			Recall()
+
+
+/mob/living/simple_animal/hostile/guardian/adjustBruteLoss(amount) //The spirit is invincible, but passes on damage to the summoner
+	var/damage = amount * damage_transfer
+	if (summoner)
+		if(loc == summoner)
+			return
+		summoner.adjustBruteLoss(damage)
+		if(damage)
+			summoner << "<span class='danger'><B>Your [name] is under attack! You take damage!</span></B>"
+			summoner.visible_message("<span class='danger'><B>Blood sprays from [summoner] as [src] takes damage!</B></span>")
+		if(summoner.stat == UNCONSCIOUS)
+			summoner << "<span class='danger'><B>Your body can't take the strain of sustaining [src] in this condition, it begins to fall apart!</span></B>"
+			summoner.adjustCloneLoss(damage/2)
+
+/mob/living/simple_animal/hostile/guardian/ex_act(severity, target)
+	switch (severity)
+		if (1)
+			gib()
+			return
+		if (2)
+			adjustBruteLoss(60)
+
+		if(3)
+			adjustBruteLoss(30)
+
+/mob/living/simple_animal/hostile/guardian/gib()
+	if(summoner)
+		summoner << "<span class='danger'><B>Your [src] was blown up!</span></B>"
+		summoner.gib()
```

Comment:
Just wanted to make sure you're ok with including this; you didn't want Dust() included--is this ok?